packages/blade/src/components/CreationView/CreationView.stories.tsx

```diff
@@ -0,0 +1,1811 @@
+                ? 'full'
+                : currentStep === step.stepNumber
+                ? 'start'
+                : 'none'
+            }
+          />
+        ))}
+      </StepGroup>
+    );
+  };
+
+  const BackdropContainer = styled.div`
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    background-color: ${({ theme }) => theme.colors.overlay.background.subtle};
+    z-index: 1004;
+  `;
+  return (
+    <Box>
+      <Button onClick={() => setIsOpen(!isOpen)}>Create GNR Details</Button>
+      {isMobile ? (
+        isOpen && (
+          <Box
+            width="100%"
+            minHeight="100%"
+            backgroundColor="surface.background.gray.moderate"
+            display="flex"
+            flexDirection="column"
+            position="fixed"
+            top="spacing.0"
+            left="spacing.0"
+            zIndex={1000}
+          >
+            {/* Header with current step name */}
+            <div
+              role="button"
+              tabIndex={0}
+              onClick={() => {
+                setShowStepGroup((prev: boolean) => !prev);
+              }}
+              // eslint-disable-next-line @typescript-eslint/no-empty-function
+              onKeyDown={() => {}}
+              style={{
+                zIndex: 1006,
+              }}
+            >
+              <Box
+                display="flex"
+                alignItems="center"
+                justifyContent="center"
+                padding="spacing.4"
+                backgroundColor="surface.background.gray.subtle"
+                borderBottomWidth="thin"
+                borderBottomColor="surface.border.gray.muted"
+                position="relative"
+              >
+                <Box display="flex" alignItems="center" gap="spacing.4">
+                  {alert?.type === 'negative' && (
+                    <InfoIcon color="feedback.icon.negative.intense" />
+                  )}
+                  <Badge color={alert?.type === 'negative' ? 'negative' : undefined}>
+                    {' '}
+                    {currentStep} / {lastStep}{' '}
+                  </Badge>
+                  <Heading size="small">{currentStepObj?.title}</Heading>
+
+                  <IconButton
+                    icon={showStepGroup ? ChevronUpIcon : ChevronDownIcon}
+                    accessibilityLabel="Open steps"
+                    onClick={() => {
+                      setShowStepGroup((prev: boolean) => !prev);
+                    }}
+                  />
+                </Box>
+              </Box>
+            </div>
+            <ProgressBar
+              value={(currentStep / lastStep) * 100}
+              showPercentage={false}
+              size="medium"
+              color={alert?.type === 'negative' ? 'negative' : undefined}
+            />
+            {showStepGroup && (
+              <Box>
+                <Slide direction="top" fromOffset="100%" motionTriggers={['mount']}>
```

Comment:
```suggestion
              <Box>
                <Slide isVisible={showStepGroup} direction="top" fromOffset="100%" motionTriggers={['mount']}>
```

Something like this would be the right usage. Can you check once in Slide docs? I don't think so exit animations will work without `isVisible` prop